cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -3691,6 +3694,7 @@ func Test_EnsureLoadBalancer(t *testing.T) {
 					},
 				},
 			},
+			wantLoadBalancerID: "other-load-balancer-id",
```

Comment:
How about `newLoadBalancerID` instead of `wantLoadBalancerID`?

Reply:
Yep, good idea -- done.